content/docs/install/access/index.md

```diff
@@ -26,28 +26,28 @@ This option exposes Keptn externally using a cloud provider's load balancer (if
 
 1. **Install Keptn:** Use the Helm CLI to install Keptn on your cluster.
 If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
-  ```console
-   helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuousDelivery.enabled=true,apiGatewayNginx.type=LoadBalancer
-  ```
-
-1. **Get Keptn endpoint:**  Get the EXTERNAL-IP of the `api-gateway-nginx` using the command below. The Keptn API endpoint is: `http://<ENDPOINT_OF_API_GATEWAY>/api`
-  ```console
-  kubectl -n keptn get service api-gateway-nginx
-  ```
-  ```console
-  NAME                TYPE        CLUSTER-IP    EXTERNAL-IP                  PORT(S)   AGE
-  api-gateway-nginx   ClusterIP   10.117.0.20   <ENDPOINT_OF_API_GATEWAY>    80/TCP    44m
-  ```
+    ```
+    helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuousDelivery.enabled=true,apiGatewayNginx.type=LoadBalancer
+    ```
+
+2. **Get Keptn endpoint:**  Get the EXTERNAL-IP of the `api-gateway-nginx` using the command below. The Keptn API endpoint is: `http://<ENDPOINT_OF_API_GATEWAY>/api`
```

Comment:
this is a markdown convention, please leave it in place. enumerations are always done with `1. `
```suggestion
1. **Get Keptn endpoint:**  Get the EXTERNAL-IP of the `api-gateway-nginx` using the command below. The Keptn API endpoint is: `http://<ENDPOINT_OF_API_GATEWAY>/api`
```